internal/parser/parser.go

```diff
@@ -71,42 +118,52 @@ func (p *Parser) ParseEntry(family, control string) error {
 
 // normalizeFamily normalizes the family name into something more
 // fitting for OpenControl.
-//
-// NOTE(jaosorior): This currently only replaces spaces for underscores...
-// we should probably replace this function with something that gets a
-// standardized name somehow
 func (p *Parser) normalizeFamily(family string) controlFamily {
-	return controlFamily(p.wre.ReplaceAllString(family, "_"))
+	// Ensure we take care of whitespace issues
+	nFamily := p.wre.ReplaceAllString(family, "_")
+	return controlFamily(parseFamily(nFamily))
 }
 
+// parseControl parses a NIST 800-53 control and ensures it conforms to the 
+// OpenControl Satisfies struct.
 func (p *Parser) parseControl(control string) (v3c.Satisfies, error) {
 	// control without extra whitespaces
 	ctrlNw := p.wre.ReplaceAllString(control, " ")
 
 	ctrl := v3c.Satisfies{}
 	if p.simpleCtrl.MatchString(ctrlNw) {
 		matches := p.simpleCtrl.FindStringSubmatch(ctrlNw)
+		fmt.Println("simpleCtrl:", matches)
```

Comment:
Did you mean these for debugging purposes? Or do you want them in the output right now?

Reply:
100% debug right now.. just makes it easier for me to see what is being parsed.